x/stakeibc/keeper/unbonding_records.go

```diff
@@ -96,6 +111,74 @@ func (k Keeper) GetTotalUnbondAmountAndRecordsIds(ctx sdk.Context, chainId strin
 	return totalUnbonded, unbondingRecordIds
 }
 
+// Update Native Amounts in the Host Zone Unbonding Record (and associated User Redemption Records) to reflect the current redemption rate
```

Comment:
A couple nit comments in this PR to clean things up a bit and keep things standardized:
* Use `.Attribute` instead of `.GetAttribute()`
* Add some vertical whitespace between logical groups of logic (makes the review much easier! - similar to new line breaks in a paragraph)
* We don't have to do that `errMsg` variable format anymore, and no need to log each error cause their logged upstream, we can just return the wrapped error